libs/openFrameworks/3d/ofNode.cpp

```diff
@@ -231,6 +231,19 @@ void ofNode::rotateAround(float degrees, const ofVec3f& axis, const ofVec3f& poi
 }
 
 //----------------------------------------
+void ofNode::lookAt(const ofVec3f& lookAtPosition){
+    auto relPosition = (getGlobalPosition() - lookAtPosition);
+    auto radius = relPosition.length();
+    if(radius>0){
+        auto latitude = ofRadToDeg(acos(relPosition.y / radius)) - 90;
+        auto longitude = ofRadToDeg(atan2(relPosition.x , relPosition.z));
```

Comment:
The only way I could see this blow up is if `relPosition.x == 0 && relPosition.y==0` - for which `atan2` is undefined. Could this ever be the case?